x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/components/constants.tsx

```diff
@@ -8,27 +8,25 @@
 import { FormattedMessage } from '@kbn/i18n-react';
 import React from 'react';
 
-export const ACTIVE_COLOR = '#E7664C';
-export const RECOVERED_COLOR = '#54B399';
 export const TOOLTIP_DATE_FORMAT = 'YYYY-MM-DD HH:mm';
 
-export const ALL_ALERT_LABEL = (
+export const WIDGET_TITLE = (
   <FormattedMessage
     id="xpack.triggersActionsUI.sections.ruleDetails.alertsSummary.title"
-    defaultMessage="Alerts"
+    defaultMessage="Alert activity"
   />
 );
 
-export const ACTIVE_ALERT_LABEL = (
+export const ALERTS_LABEL = (
   <FormattedMessage
-    id="xpack.triggersActionsUI.sections.ruleDetails.alertsSummary.activeLabel"
-    defaultMessage="Active"
+    id="xpack.triggersActionsUI.sections.ruleDetails.alertsSummary.alerts"
+    defaultMessage="Alerts"
   />
 );
 
-export const RECOVERED_ALERT_LABEL = (
+export const ACTIVE_ALERT_LABEL = (
   <FormattedMessage
-    id="xpack.triggersActionsUI.sections.ruleDetails.rule.ruleSummary.recoveredLabel"
-    defaultMessage="Recovered"
+    id="xpack.triggersActionsUI.sections.ruleDetails.alertsSummary.activeNow"
+    defaultMessage="Active Now"
```

Comment:
We usually use sentence case in Kibana. cc @maciejforcone 

```suggestion
    defaultMessage="Active now"
```